src/Drupal/Commands/core/FieldCreateCommands.php

```diff
@@ -26,7 +26,9 @@
 
 class FieldCreateCommands extends DrushCommands implements CustomEventAwareInterface
 {
+    use AskBundleTrait;
     use CustomEventAwareTrait;
```

Comment:
Looks like we don't use a CustomEvent directly.

Reply:
We do call the `getCustomEventHandlers` method in `FieldCreateCommands`, right?